.pre-commit-config.yaml

```
@@ -35,36 +35,26 @@ repos:
         # The pre-commit hook only has python, not pyi.
         types: []
         types_or: [python, pyi]
-        # TODO: unify the versions once Python 3.8 support is dropped
         additional_dependencies: [
-          'numpy==1.24.4; python_version<"3.12.0rc1"',
-          'numpy==2.0.0; python_version>="3.12.0rc1"'
+          'numpy==2.0.2'
         ]
   - repo: https://github.com/jazzband/pip-tools
-    rev: 7.1.0
+    rev: 7.4.1
     hooks:
-      - name: pip-compile (py 3.8)
+      - name: pip-compile (py 3.9)
         id: pip-compile
         args: [requirements.in]
-        language_version: python3.8
+        language_version: python3.9
         files: '^requirements\.(?:in|txt)$'
-  - repo: https://github.com/jazzband/pip-tools
-    rev: 7.4.1
-    hooks:
-      - name: pip-compile (py 3.12)
-        id: pip-compile
-        args: [--strip-extras, requirements.in, -o, requirements-3.12.txt]
-        language_version: python3.12
-        files: '^(?:requirements\.in|requirements-3\.12\.txt)$'
       - name: pip-compile (py 3.13)
         id: pip-compile
         args: [--strip-extras, requirements.in, -o, requirements-3.13.txt]
         language_version: python3.13
         files: '^(?:requirements\.in|requirements-3\.13\.txt)$'
       - name: pip-compile (readthedocs)
         id: pip-compile
-        # --no-annotation because otherwise -r requirements-3.12.txt sometimes gets
+        # --no-annotation because otherwise -r requirements.txt sometimes gets
```

```suggestion
        # --no-annotate because otherwise -r requirements.txt sometimes gets
```
nit-picking, not that it makes a difference

---

Thanks for catching it.